modules/post/linux/gather/enum_nagios_xi.rb

```diff
@@ -1,132 +1,137 @@
-  ##
-  # This module requires Metasploit: https://metasploit.com/download
-  # Current source: https://github.com/rapid7/metasploit-framework
-  ##
-
-  class MetasploitModule < Msf::Post
-    include Msf::Post::Linux::System
-    include Msf::Exploit::FileDropper
-
-    def initialize(info={})
-      super(update_info(info, {
-        'Name'           => 'Nagios XI Enumeration',
-        'Description'    => %q{
-          NagiosXI may store credentials of the hosts it monitors. This module extracts these credentials,
-          creating opportunities for lateral movement.
-        },
-        'License'        => MSF_LICENSE,
-        'Author'         =>         [
-            'Cale Smith',   # @0xC413
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Linux::System
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        {
+          'Name' => 'Nagios XI Enumeration',
+          'Description' => %q{
+            NagiosXI may store credentials of the hosts it monitors. This module extracts these credentials,
+            creating opportunities for lateral movement.
+          },
+          'License' => MSF_LICENSE,
+          'Author' => [
+            'Cale Smith', # @0xC413
           ],
-        'DisclosureDate'  => '2018-04-17',
-        'Platform'       => 'linux',
-        'SessionTypes'   => ['shell', 'meterpreter'],
+          'DisclosureDate' => '2018-04-17',
+          'Platform' => 'linux',
+          'SessionTypes' => ['shell', 'meterpreter']
         }
-      ))
-      register_options([
-        OptString.new('DB_ROOT_PWD', [true, 'Password for DB root user, an option if they change this', 'nagiosxi' ])
-      ])
-    end
-
-    # save found creds in the MSF DB for easy use
-    def report_obj(cred, login)#, login)
-      return if cred.nil? || login.nil?
-      credential_data = {
-        origin_type: :session,
-        post_reference_name: self.fullname,
-        session_id: session_db_id,
-        workspace_id: myworkspace_id,
-
-      }.merge(cred)
-      credential_core = create_credential(credential_data)
+      )
+      )
+    register_options([
+      OptString.new('DB_ROOT_PWD', [true, 'Password for DB root user, an option if they change this', 'nagiosxi' ])
+    ])
+  end
 
-      login_data = {
-        core: credential_core,
-        workspace_id: myworkspace_id
-      }.merge(login)
+  # save found creds in the MSF DB for easy use
+  # , login)
```

Comment:
uhhh weird comment, probably should be deleted rather than moved to it's own line but whatever